libs/design/select/examples/src/default-select/default-select.component.html

```diff
@@ -0,0 +1,26 @@
+<daff-select [options]="options" [formControl]="control">
+  <ng-template daffSelectSelectedOption let-selected="selected">
+    <div *ngIf="!selected">
+      Select an address
+    </div>
+    <div *ngIf="selected">
+      <div>{{selected.name}}</div>
+      <div>{{selected.street}}</div>
+      <div>{{selected.city}}, {{selected.state}} {{selected.postcode}}</div>
+    </div>
+  </ng-template>
+  <ng-template daffSelectOption let-isSelected="isSelected" let-option="option">
+    <div [ngStyle]="{color: isSelected ? 'green' : 'initial'}">
```

Comment:
This selected style should be defined in the select component, not in the implementation.